packages/react/src/table/Table.test.tsx

```diff
@@ -21,12 +21,12 @@ afterEach(cleanup);
 function renderTableWithCaption() {
 	return render(
 		<TableWrapper>
-			<Table id="table">
+			<Table id="table" aria-rowcount={9}>
```

Comment:
This file can be reverted. You only need to use `aria-rowcount` and `aria-rowindex` when there is paginated data. 

> If all of the rows are loaded and in the DOM, you don't need to include aria-rowcount as browsers automatically count the total number of rows.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-rowcount